grandeur/grandeur_results_files.py

```diff
@@ -0,0 +1,311 @@
+#!/usr/bin/env python3
+
+'''
+Author: Erin Young
+
+Description:
+
+This script will collect needed files to get results from Grandeur for 
+1. The 'Finished' tab (will be discontinued in the future)
+2. As a lookup table for the 'ARLN' tab
+3. As input for Labware 8 for 
+  a. ALT-NGS
+  b. ARLN samples
+  c. GC samples
+  d. pulsenet samples
+
+(Yes, that is potentially seven files)
+
+EXAMPLE:
+python3 grandeur_results_files.py -g <path to grandeur results> -s <input sample sheet>
```

Comment:
This script is suppose to be ran from run directory? Like /Volumes/IDGenomics_NAS/pulsenet_and_arln/UT-M07101-231003.